crates/holochain/tests/tests/conductor/mod.rs

```diff
@@ -0,0 +1,403 @@
+use std::collections::HashMap;
```

Comment:
I suggest to add a comment at the beginning of the `app_impl` block in `conductor.rs` that links to this test module.